src/style.css

```diff
@@ -0,0 +1,101 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.flex {
+  height: 300px;
+  min-width: 300px;
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+  text-align: center;
+  line-height: 300px;
+  flex-grow: 1;
+  flex-basis: 33%;
+}
+
+.box_1 {
+  background-color: hsl(0,100%,50%);
+  order: 1;
+}
+
+.box_2 {
+  background-color: hsl(0,100%,40%);
+  order: 2;
+}
+
+.box_3 {
+  background-color: hsl(0,100%,30%);
+  order: 3;
+}
+
+.box_4 {
+  background-color: hsl(0,100%,20%);
+  order: 4;
+}
+
+.box_5 {
+  background-color: hsl(0,100%,10%);
+  order: 5;
+}
+
+.box_6 {
+  background-color: hsl(0,0%,0%);
+  order: 6;
+}
+
+@media screen and (min-width: 916px) {
+  .box_4 {
+    order: 6;
+  }
+  .box_6 {
+    order: 4;
+  }
+}
+
+@media screen and (min-width: 617px) and (max-width: 916px) {
+  .box_1 {
+    order: 1;
+  }
+  .box_2 {
+    order: 2;
+  }
```

Comment:
in media queries write only those properties, that should be changed. Omit those, that stays the same